app/assets/stylesheets/comments.scss

```diff
@@ -53,12 +55,14 @@ a.header-link {
     margin: 50px auto 10px;
     padding: 0px 0px 90px;
     background: white;
+    background: var(--theme-container-background, #fff);
```

Comment:
Even if it is the same value isn't it better to keep the variable `white` for consistency? Just a personal taste.

Reply:
Sure, no problem.